_docs-sources/iac/whats-this/modules.md

```diff
@@ -1,11 +1,21 @@
 # What is a Module?
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Est placerat in egestas erat imperdiet sed. Velit dignissim sodales ut eu sem integer vitae. Mattis rhoncus urna neque viverra. Massa placerat duis ultricies lacus sed turpis tincidunt id. Posuere sollicitudin aliquam ultrices sagittis orci. Egestas quis ipsum suspendisse ultrices gravida dictum fusce ut placerat. Tristique senectus et netus et malesuada. Ipsum dolor sit amet consectetur adipiscing elit pellentesque. Enim nunc faucibus a pellentesque sit amet porttitor eget dolor. Eget nunc lobortis mattis aliquam faucibus purus in. Vulputate dignissim suspendisse in est ante in nibh mauris. Morbi non arcu risus quis varius quam. Cras adipiscing enim eu turpis egestas pretium. Donec enim diam vulputate ut. Ut tortor pretium viverra suspendisse potenti nullam. Euismod elementum nisi quis eleifend quam adipiscing vitae proin sagittis. Sed nisi lacus sed viverra tellus. Laoreet id donec ultrices tincidunt arcu non sodales neque.
+Modules are reusable code to deploy and manage one piece of infrastructure and each one is a battle-tested, best-practices definition of a piece of infrastructure, such as a VPC, ECS cluster, or an Auto Scaling Group.
 
-## Whats a Module
+## When do you use a Module?
 
-It's a bunch of code, ya silly!
+Modules are fairly generic building blocks, so you don't typically deploy a single module directly, but rather, you write code that combines the modules you need for a specific use case. 
 
-## When do you use a Module?
+For example, one module might deploy the control plane for Kubernetes and a separate module could deploy worker nodes; you may need to combine both modules together to deploy a Kubernetes cluster, or use the `eks-cluster` service from the [Service Catalog](/iac/whats-this/services/).
+
+The Gruntwork Infrastructure as Code (IaC) Library contains hundreds of battle-tested, commercially supported and maintained modules that you can use and combine in many different ways. Modules are versioned using Semantic Versioning to allow Gruntwork clients to keep up to date with the latest infrastructure best practices in a systematic way.
```

Comment:
This feels better suited for a "what's next" section. Maybe something like..

```
The Gruntwork Infrastructure as Code (IaC) Library contains hundreds of modules that you can use and combine in many different ways. For a full list of modules available, refer to the [Library Reference](link to lib reference index).
```

I think we can strike the versioning bit, it's covered in the versioning page.